code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -200,6 +218,15 @@
 	use_power(power_usage)
 	update_parents()
 
+/obj/machinery/atmospherics/components/unary/thermomachine/attackby(obj/item/assembly/signaler/anomaly/pyro, mob/living/user, params)
+	if(!has_pyro_anomaly_core && allows_cooling)
+		to_chat(user, span_notice("The anomaly core fits perfectly!"))
```

Comment:
```suggestion
		balloon_alert(user, "inserted anomaly core")
```

Reply:
Changed and tested it. It does look better this way